### PR TITLE
Prettify network-mux diagrams

### PR DESCRIPTION
**Pull request by coot**
Include ingress diagram in haddock.

### CODE REVIEW

File: network-mux/src/Network/Mux/Ingress.hs [hunk 1]
```diff
@@ -28,43 +29,46 @@ negMiniProtocolMode ModeResponder = ModeInitiator
...
+-- >         ░░░░░░░░░▼░░░░░░░░░
+-- >         ░┌───────────────┐░
+-- >         ░│ Bearer.read() │░ Mux Bearer implementation (Socket, Pipes, etc.)
+-- >         ░└───────────────┘░
+-- >         ░░░░░░░░░│░░░░░░░░░
+-- >                 ░│░         MuxSDUs
+-- >         ░░░░░░░░░▼░░░░░░░░░
+-- >         ░┌───────────────┐░
+-- >         ░│     demux     │░ For a given Mux Bearer there is a single demux
+-- >         ░└───────┬───────┘░ thread reading from the underlying be arer.
```
Comment (karknu) [inline network-mux/src/Network/Mux/Ingress.hs]: s/be arer/bearer

File: network-mux/src/Network/Mux/Ingress.hs [hunk 2]
```diff
@@ -28,43 +29,46 @@ negMiniProtocolMode ModeResponder = ModeInitiator
...
+-- >         ░░░░░░░░░│░░░░░░░░░
+-- >                 ░│░
+-- >        ░░░░░░░░░░▼░░░░░░░░░░
+-- >        ░ ╭────┬────┬─────╮ ░ There is a limited queue (in bytes) for each mode
+-- >        ░ │    │    │     │ ░ (responder/initiator) perminiprotocol. Overflowing
```
Comment (karknu) [inline network-mux/src/Network/Mux/Ingress.hs]: s/perminiprotocol/per miniprotocol

File: network-mux/src/Network/Mux/Ingress.hs (hunk 1 above)
Comment (coot) [reply network-mux/src/Network/Mux/Ingress.hs]: Ha ha :)

Comment (coot) [reply network-mux/src/Network/Mux/Ingress.hs]: Fixed

File: network-mux/src/Network/Mux/Ingress.hs (hunk 2 above)
Comment (coot) [reply network-mux/src/Network/Mux/Ingress.hs]: Fixed

**Pull request merged**